packages/sdk/src/controllers/UndoManagerController.ts

```diff
@@ -41,6 +41,19 @@ export class UndoManagerController {
         return res.redo().then((result) => getEditorResponseData<null>(result));
     };
 
+    /**
+     * This method adds custom data that will be saved and restored when undoing and redoing.
+     * Duplicate values are overwritten. The data is exposed via the onCustomUndoDataChanged event.
+     * 
+     * @param key The key of the custom data
+     * @param value The value of the custom data
+     * @returns 
+     */
+    addCustomData = async (key: string, value: string) => {
```

Comment:
Didn't we forget to add `getCustomData`? Not every integration is going to use the events.